sites/svelte.dev/src/routes/(authed)/repl/[id]/+page.server.js

```diff
@@ -1,5 +1,7 @@
 import { error } from '@sveltejs/kit';
 
+// export const prerender = 'auto';
+
```

Comment:
guessing this was left over from testing?

```suggestion
```

Reply:
Removed